src/core/openthread.cpp

```diff
@@ -876,6 +878,33 @@ ThreadError otRemoveUnicastAddress(otInstance *, const otIp6Address *address)
     return sThreadNetif->RemoveExternalUnicastAddress(*static_cast<const Ip6::Address *>(address));
 }
 
+void otSlaacUpdate(otInstance *aInstance, otNetifAddress *aAddresses, uint32_t aNumAddresses,
+                   otSlaacIidCreate aIidCreate, void *aContext)
+{
+    Utils::Slaac::UpdateAddresses(aInstance, aAddresses, aNumAddresses, aIidCreate, aContext);
+}
+
+ThreadError otCreateRandomIid(otInstance *aInstance, otNetifAddress *aAddress, void *aContext)
+{
+    return Utils::Slaac::CreateRandomIid(aInstance, aAddress, aContext);
+}
+
+ThreadError otCreateMacIid(otInstance *aInstance, otNetifAddress *aAddress, void *aContext)
+{
+    (void) aInstance;
+    (void) aContext;
```

Comment:
Since this is C++ code, I'd prefer to just remove the variable name from the  declaration instead of having the (void) ...; But it's just a suggestion.